css/querys.css

```diff
@@ -1,316 +1,318 @@
+    grid-template-rows: repeat(6, 435px);
+    grid-row-gap: 10px;
+    margin-top: 10px;
+    margin-bottom: 100px;
+  }
+
+  .about-me-bg {
+    min-height: 700px;
+    background-image: url("../images/footer-bg.png");
+    background-position: right bottom, left top;
+    background-repeat: no-repeat, repeat;
+    background-size: 120px 210px;
+    transition: background 3s;
+  }
+
+  .about-me {
+    width: 100%;
+  }
+
+  .f-lang {
+    padding: 80px 0;
+    margin: 30px 0;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .languages {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    min-width: 350px;
+  }
+
+  .languages ul li {
+    display: flex;
+    justify-content: center;
+  }
+
+  .frameworks {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    min-width: 350px;
+  }
+
+  .frameworks ul li {
+    display: flex;
+    justify-content: center;
+  }
+
+  .skills {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    min-width: 350px;
+  }
+
+  .skills ul li {
+    display: flex;
+    justify-content: center;
+  }
+
+  .contact-form-main {
+    display: flex;
+    flex-direction: column;
+    padding: 0 10%;
+    width: 100%;
+    margin-top: 100px;
+  }
+
+  input[name="name"],
+  input[name="email-address"] {
+    width: 100%;
+    height: 48px;
+  }
+
+  .cart-1 {
+    width: 100%;
+    height: 435px;
+    background-image: url("../images/card-1.png");
+    background-repeat: no-repeat, repeat;
+    border: 1px solid #000;
+  }
+
+  .card-btn {
+    height: 58px;
+    width: 100%;
+    background: #f55800;
+    color: #fff;
+    border: none;
+    font-family: 'Inter', sans-serif;
+    font-size: 17px;
+    font-weight: 700;
+    text-align: center;
+    margin-top: 377px;
+  }
+
+  .card-btn:hover {
+    height: 58px;
+    width: 100%;
+    background: #fff;
+    color: #f55800;
+    border: 1px solid #f55800;
+    font-family: 'Inter', sans-serif;
+    font-size: 17px;
+    font-weight: 700;
+    text-align: center;
+    margin-top: 377px;
+  }
+
+  .cart-2 {
+    width: 100%;
+    height: 435px;
+    background-image: url("../images/card-2.png");
+    background-repeat: no-repeat, repeat;
+  }
+
+  .cart-2 h2 {
+    font-family: 'Crete Round', serif;
+    font-size: 32px;
+    font-weight: 400;
+    color: #fff;
+    padding-left: 10px;
+    margin-top: 180px;
+  }
+
+  .cart-2 p {
+    font-family: 'Inter', sans-serif;
+    font-size: 17px;
+    font-weight: 400;
+    color: #fff;
+    padding-left: 10px;
+    margin-top: 10px;
+  }
+
+  .cart-3 {
+    width: 100%;
+    height: 435px;
+    background-image: url("../images/card-3.png");
+    background-repeat: no-repeat, repeat;
+  }
+
+  .cart-3 h2 {
+    font-family: 'Crete Round', serif;
+    font-size: 32px;
+    font-weight: 400;
+    color: #fff;
+    padding-left: 10px;
+    margin-top: 210px;
+  }
```

Comment:
# 💾 💾 💽 [OPTIONAL]   💽 💾 💾
**✔️ _Improve your ``CSS`` files_** 
- ⏏︎ Use the DRY method to keep your projects optimized and clean.

If you want to learn more about this [HERE](https://vanseodesign.com/css/dry-principles/).